shuffler/deployment.yaml

```diff
@@ -48,10 +48,10 @@ spec:
              readOnly: true
           resources:
             requests:
-              memory: "3.75G"
-              cpu: "800m"
+              memory: 375Mi
```

Comment:
This seems tight.  Shuffler is currently at 343Mi.

Reply:
It seems we should set the request to the idle number. If a node is overloaded, my understanding, is that k8s will kill off and reschedule pods exceeding their requests. Ideally this moves the shuffler to a less loaded node.

If we see high turn over for shuffler, I'll adjust the requests then.